crates/winmd/src/types/delegate.rs

```diff
@@ -105,6 +105,15 @@ impl Delegate {
                     self.ptr.set_abi()
                 }
             }
+            impl<#constraints> ::std::fmt::Debug for #name {
+                fn fmt(&self, f: &mut ::std::fmt::Formatter<'_>) -> ::std::fmt::Result {
+                    write!(
+                        f,
+                        "{:?}",
```

Comment:
This doesn't seem too helpful. Shouldn't we also print the name of the type as well?

Reply:
The was primarily for completeness to unblock the issues with importing WinRT modules that used WinRT structs that have interface fields. WinRT structs mainly have primitive value fields and its very useful to have the Debug trait for those, but there are a handful that also have interface fields. I created https://github.com/microsoft/winrt-rs/issues/138 to address making these more helpful.